CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v1.33.0
+### Notable Changes
+* Migrate CreateVolume response topology to standard label topology.kubernetes.io/zone ([#2086](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2086), [@ConnorJC3](https://github.com/ConnorJC3))
+* Modifying Tags On Existing Volume ([#2082](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2082), [@mdzraf](https://github.com/mdzraf))
```

Comment:
Suggestion: Rename to "Add ability to modify EBS volume tags via VolumeAttributesClass"